.github/workflows/moodle-ci.yml

```diff
@@ -53,6 +56,7 @@ jobs:
           sudo apt-get install gnuplot maxima maxima-share texinfo
           maxima --list-avail
           echo "diff(x^2,x);" | maxima
+          echo "bug_report();" | maxima
```

Comment:
What does this do?

Reply:
It outputs Maxima build info; something like this:

```
Maxima 5.41.0 http://maxima.sourceforge.net
using Lisp SBCL 1.3.14.debian
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1)
Please report bugs to:
    https://sourceforge.net/p/maxima/bugs
To report a bug, you must have a Sourceforge account.
Please include the following information with your bug report:
-------------------------------------------------------------
Maxima version: "5.41.0"
Maxima build date: "2017-11-01 10:23:09"
Host type: "x86_64-pc-linux-gnu"
Lisp implementation type: "SBCL"
Lisp implementation version: "1.3.14.debian"
-------------------------------------------------------------
The above information is also reported by the function 'build_info()'.
```

It's useful to have this information to compare in the case that unit tests are failing locally but not in GHA (or vica-versa) to rule out mistmatched builds.

I think it would probably be better to use `build_info` though - as it's clearer from the name what it does.